src/test/mod.rs

```diff
@@ -297,6 +297,46 @@ fn test_reformat() {
 }
 
 #[test]
+fn test_reformat_with_range() {
+    let (cache, _tc) = init_env("reformat");
```

Comment:
You'll need to duplicate the test_data directory so each test is pointing at its own directory (since tests run concurrently, they'll otherwise race)